### PR TITLE
Add possibility to mark tag as asset tag

### PR DESCRIPTION
**Pull request by jitka1997**

### CODE REVIEW

File: lib/addTransaction/state.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface Tag extends FirebaseField {
   name: string
   automatic: boolean
   defaultAmount?: string // can be empty
+  isAsset?: boolean // older tags have this empty
```
Comment (Siegrift) [inline lib/addTransaction/state.ts:10]:
```suggestion
  isAsset?: boolean // older tags do not have this property defined
```

File: lib/tags/tagDetails.tsx [hunk 2]
```diff
@@ -178,6 +181,28 @@ const TagDetails = ({ tag, stats, appBarTitle, onSave, onRemove }: TagDetailsPro
...
             }
           />
+          <FormControlLabel
+            classes={{ label: classes.label }}
+            style={{ flex: 1, width: '100%' }}
+            control={
+              <Checkbox
+                checked={isAsset}
+                onChange={() => setIsAsset(!isAsset)}
+                value="checkedA"
```
Comment (Siegrift) [inline lib/tags/tagDetails.tsx:191]: Can you remove this `value` and `inputProps` (from other component as well). They are extraneous.

File: lib/tags/tagItem.tsx [hunk 3]
```diff
@@ -80,6 +81,11 @@ const TagItem: React.FC<ListChildComponentProps> = ({ index, style }) => {
                 <AutoIcon className={classes.icon} />
               </Tooltip>
             )}
+            {tag.isAsset && (
+              <Tooltip title="In asset">
```
Comment (Siegrift) [inline lib/tags/tagItem.tsx:85]:
```suggestion
              <Tooltip title="Is asset">
```

**Review by Siegrift — CHANGES_REQUESTED** [review]

File: lib/tags/tagItem.tsx (hunk 3 above)
Comment (jitka1997) [reply lib/tags/tagItem.tsx:85]: I don't think "Is asset" is a better title, because the tag isn't technically an asset

Comment (Siegrift) [reply lib/tags/tagItem.tsx:85]: Yeah, lets do `Represents asset` since "in asset" also doesn't make much sense. And asset tags will be very special.

**Review by Siegrift — APPROVED** [review]
:1st_place_medal: LGTM

**Pull request merged**